src/components/v5/common/ActionSidebar/partials/hooks.ts

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from 'react';
+import { Actions } from '~constants/actions';
+import { useActionSidebarContext } from '~context/ActionSidebarContext';
+
+export const useGetSubmitButton = () => {
+  const { selectedAction } = useActionSidebarContext();
+  const [submitButtonText, setSubmitButtonText] = useState(
+    'button.createAction',
+  );
+
+  useEffect(() => {
+    switch (selectedAction) {
```

Comment:
please add type and try to use `satisty never` like Mat https://www.youtube.com/channel/UCswG6FSbgZjbWtdf_hMLaow/community?lb=UgkxDNktFDKfbg0vwLwNkctzsIJnEVu85Im6

default your_type satisty never